website/src/content/blog/2025-03-31-typespec-1-0-release/typespec_1_0_release.md

```diff
@@ -0,0 +1,215 @@
+---
+slug: 2025-03-31-typespec-1-0-release
+title: "TypeSpec 1.0-RC: Write your API once, let TypeSpec do the rest"
+image: ./TypeSpec_1_0_RC.png
+description: "The TypeSpec 1.0 Release Candidate is here! This open-source project, created by Microsoft, brings a powerful yet concise API modeling language that generates server code, client libraries, schemas, and documentation from a single source of truth."
+publishDate: 2025-03-19
+author:
+  name: Mario Guerra
+  title: Senior Product Manager @ Microsoft
+authorAvatar: assets/img/authors/mario_guerra.png
+tags:
+  - release
+  - announcement
+---
+
+We're excited to announce the TypeSpec 1.0 Release Candidate, marking a significant milestone for the TypeSpec project. TypeSpec embodies a simple yet powerful principle: write your API once, and let TypeSpec take care of the rest. This straightforward approach allows you to focus on designing the perfect API contract while TypeSpec handles generating all the artifacts you need - from OpenAPI specifications to client libraries and server code scaffolding - saving you time and eliminating inconsistencies.
```

Comment:
It might be stronger and more compelling to leave out the "write your API once". In reality, API evolve over time and TypeSpec, helps with this evolution. So, you might want to address this.

Reply:
Fair point, but I do want to emphasize that using TypeSpec will save time. How about something that emphasizes an API-First approach like "start from an API definition and let TypeSpec take care of the rest"